modules/apps/adaptive-media/adaptive-media-web/src/main/java/com/liferay/adaptive/media/web/internal/servlet/AMServlet.java

```diff
@@ -100,6 +102,19 @@ protected void doGet(
 
 			String fileName = fileNameOptional.orElse(null);
 
+			String uri = String.valueOf(adaptiveMedia.getURI());
```

Comment:
All this can be done OR before the declaration of `contentLengthOptional` OR before the declaration of `download` to do de declaration of the variables  near the use.